controller/k8s/api.go

```diff
@@ -473,6 +476,11 @@ func (api *API) GetPodsFor(obj runtime.Object, includeFailed bool) ([]*corev1.Po
 		selector = labels.Set(typed.Spec.Selector.MatchLabels).AsSelector()
 		ownerUID = typed.UID
 
+	case *v1alpha1.TrafficSplit:
```

Comment:
Why do we need to implement GetPodsFor for TrafficSplits?  I'm not sure that makes sense...